corehq/apps/styleguide/templates/styleguide/bootstrap5/htmx_and_alpine.html

```diff
@@ -49,240 +63,583 @@ <h2 id="overview" class="pt-4">
+  <ul>
+    <li>
+      A single <strong>host view</strong> renders the overall page layout, navigation,
+      common context, and JavaScript entry point.
+    </li>
+    <li>
+      Several <strong>section views</strong> each manage one part of the page.
+      Each section view has its own URL and its own HTMX actions.
+    </li>
+  </ul>
+  <p>
+    The host view passes section view URLs into the template via context, and the
+    template uses those URLs as <code>hx-get</code>/<code>hx-post</code> targets for
+    different parts of the page.
   </p>
-  {% include 'styleguide/bootstrap5/code_display.html' with content=examples.htmx_hq_hx_action %}
+
+  <h3 id="bulk-edit-cases" class="pt-3">
+    Example: Bulk Edit Cases (Host View + Section Views)
+  </h3>
   <p>
-    Looking at the main template for this view, we can scan the page for the first <code>hx-post</code>
-    attribute, which posts to <code>request.path_info</code>. This is URL of the page serving this
-    template: <code>TodoListDemoView</code>.
+    A concrete production example of this pattern is the bulk data editing UI in
+    <a
+      href="https://github.com/dimagi/commcare-hq/tree/master/corehq/apps/data_cleaning"
+      target="_blank"
+    ><code>corehq.apps.data_cleaning</code></a>. The
+    <a
+      href="https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_cleaning/views/main.py#L60"
+      target="_blank"
+    ><code>BulkEditCasesSessionView</code></a> acts as the host view for the page, and several
+    section views handle individual areas of the UI.
   </p>
+
   <p>
-    Below <code>hx-post</code> is the <code>hq-hx-action</code> attribute referencing the
-    <code>create_new_item</code> method of the <code>TodoListDemoView</code>. Remember, this was one
-    of the methods decorated with <code>@hq_hx_action()</code>. The rest of the attributes are specific to
-    HTMX, and you can read more about them on <a href="https://htmx.org/" target="_blank">htmx.org</a>.
+    In
+    <a
+      href="https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_cleaning/views/main.py#L104-L130"
+      target="_blank"
+    ><code>BulkEditCasesSessionView.page_context</code></a>, the host view exposes the URLs
+    of its section views.
   </p>
+
   <p>
-    If you scan the rest of the template, you will notice another <code>hq-hx-action</code> attribute which
-    references the method <code>does_not_exist</code>. As it is aptly named, this method does not exist in
-    <code>TodoListDemoView</code>. However, that's the point, as this attribute is attached to a button which
-    triggers the HTMX error modal. :)
-  </p>
-  {% include 'styleguide/bootstrap5/code_display.html' with content=examples.htmx_todo_main %}
+    Each of these URLs points to a section view responsible for one part of the page:
+  </p>
+  <ul>
+    <li>
+      <a
+        href="https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_cleaning/views/tables.py#L40"
```

Comment:
Consider using a commit hash instead of `master`. (fwiw, master is currently 5c8ae80653a )